src/EventStore.Core/Services/Transport/Grpc/Operations.Scavenge.cs

```diff
@@ -46,36 +47,31 @@ partial class Operations {
 				ScavengeId = scavengeId,
 				ScavengeResult = scavengeResult
 			};
-
+			
 			void OnMessage(Message message) => HandleScavengeDatabaseResponse(message, scavengeResultSource);
 		}
 
 		private static void HandleScavengeDatabaseResponse(
 			Message message,
 			TaskCompletionSource<(string, ScavengeResp.Types.ScavengeResult)> scavengeResultSource) {
-			if (!(message is ClientMessage.ScavengeDatabaseResponse response)) {
-				scavengeResultSource.TrySetException(
```

Comment:
intentionally dropped the unknown message handling?

Reply:
@timothycoleman In the original code, we have one message type ClientMessage.ScavengeDatabaseResponse and now we have various message types. What would be the ideal way to handle unknown message in this scenario?

Reply:
fab :+1: